### PR TITLE
engine: print stdout from docker

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: internal/build/docker.go [hunk 1]
```diff
@@ -375,6 +374,9 @@ func readDockerOutput(reader io.Reader) (*json.RawMessage, error) {
 			return nil, fmt.Errorf("decoding docker output: %v", err)
 		}
 
+		// TODO(Han): make me smarter! 🤓
+		stdout.Write([]byte(fmt.Sprintf("%+v\n", message)))
```
Comment (jazzdan) [inline internal/build/docker.go]: We pass the `io.Writer` all the way through, just to get it here and print out this ugly message. @yuindustries is going to make the output a lot better in the future, but for now we can see what's going on.

**Comment by nicks** [comment]
hmmm...i thought the goal of @landism 's changes were to move away from having io.Writers and towards using context-attached loggers for output?

File: internal/build/docker.go [hunk 2]
```diff
@@ -391,7 +396,7 @@ func readDockerOutput(reader io.Reader) (*json.RawMessage, error) {
 }
 
 func getDigestFromBuildOutput(reader io.Reader) (digest.Digest, error) {
-	aux, err := readDockerOutput(reader)
+	aux, err := readDockerOutput(reader, os.Stdout)
```
Comment (nicks) [inline internal/build/docker.go]: hmmm...this seems wrong?

**Comment by jazzdan** [comment]
Yeah closing in favor of https://github.com/windmilleng/tilt/pull/117

**Pull request closed without merging**